src/dom/context2d.js

```diff
@@ -1,22 +1,20 @@
 export default function(width, height, options) {
-  var colorSpace = "srgb";
   var scale = devicePixelRatio;
-  if (options == +options) {
+  if (options == null) {
+    options = {};
```

Comment:
It’d be better to be strict here and only promote _options_ to an object when it is undefined, ideally using default arguments (`options = {}`). If the user passes in null explicitly, we should respect that and pass null to canvas.getContext.